app/controllers/bz_controller.rb

```diff
@@ -16,27 +20,36 @@ def last_user_url
 
   def video_link
     obj = {}
-    # For PoC I am using a manually generated list of links
-    # if this passes the test i will make it automatic via
-    # braven.developer@gmail.com
-    links = [
-      'https://talkgadget.google.com/hangouts/_/h5uscfakbtmoq4upfj3vqudx5ya?hl=en&authuser=0',
-      'https://talkgadget.google.com/hangouts/_/33mbnww6gddxvidixwbiirf2aaa?hl=en&authuser=0',
-      'https://talkgadget.google.com/hangouts/_/mcspdwadurnln2lapwourzeanaa?hl=en&authuser=0',
-      'https://talkgadget.google.com/hangouts/_/bovr3tn7z74jlo5jvkdagzixlaa?hl=en&authuser=0',
-      'https://talkgadget.google.com/hangouts/_/vvycnh3topzabunpctvyzdrshea?hl=en&authuser=0',
-      'https://talkgadget.google.com/hangouts/_/kphma7mmdlqpletyg3mepnq3tia?hl=en&authuser=0',
-      'https://talkgadget.google.com/hangouts/_/bb6frjablw3d246ejqna7umz74a?hl=en&authuser=0',
-      'https://talkgadget.google.com/hangouts/_/2ndb54z2i5hwniaz44czyg77gea?hl=en&authuser=0',
-      'https://talkgadget.google.com/hangouts/_/s2utjhxjoxpaksd5vgmytwdljya?hl=en&authuser=0',
-      'https://talkgadget.google.com/hangouts/_/c3o3zep5bciimgxshywuxbnujea?hl=en&authuser=0',
-      'https://talkgadget.google.com/hangouts/_/ruqgm7nkczglgqtzgeaweqa574a?hl=en&authuser=0',
-      'https://talkgadget.google.com/hangouts/_/y5swzttonly63hgjgzi3yob3zaa?hl=en&authuser=0',
-      'https://talkgadget.google.com/hangouts/_/5hhyqhtv24p3ghd2x2muevuckma?hl=en&authuser=0',
-      'https://talkgadget.google.com/hangouts/_/gh5qxhjongpvbcv2lwlct7xsima?hl=en&authuser=0',
-      'https://talkgadget.google.com/hangouts/_/5d4oidzr7b4z3uyvd5bvtge3taa?hl=en&authuser=0'
-    ]
-    obj['link'] = links.sample # in the real thing, we will use a database so we never reuse an item either but here it will just be a random one
+
+    client = Google::APIClient.new(:application_name => 'Braven Canvas')
+
+    file_store = Google::APIClient::FileStore.new(File.join(Rails.root, "config", "calendar-ruby-quickstart.json"))
+    storage = Google::APIClient::Storage.new(file_store)
+    client.authorization = storage.authorize
+    calendar_api = client.discovered_api('calendar', 'v3')
+
+    event = {
+      'summary' => 'Canvas event',
+      'start' => {
+        'dateTime' => DateTime.now.iso8601,
+        'timeZone' => 'America/Los_Angeles',
+      },
+      'end' => {
+        'dateTime' => (DateTime.now + 1.hours).iso8601,
```

Comment:
On Fri, Oct 30, 2015 at 08:10:19AM -0700, Brian Sadler wrote:
> Does this mean that they can't setup events for longer than 1 hour?  I guess is doesn't matter b/c it's behind the scenes and google doesn't kick you off the hangout when the meeting ends.  Same thing for the timezone affecting the start time.  You just need to get the link, but having the event at the exact right time on the google cal doesn't matter.

Yeah, this is a required field by the calendar api but it isn't
actually used at all.